xikolo-common/ViewHelpers/CollectionViewMultipleResultsControllerDelegateImplementation.swift

```diff
@@ -0,0 +1,144 @@
+        contentChangeOperations.removeAll()
+    }
+
+    func controller(_ controller: NSFetchedResultsController<NSFetchRequestResult>, didChange sectionInfo: NSFetchedResultsSectionInfo, atSectionIndex sectionIndex: Int, for type: NSFetchedResultsChangeType) {
+        contentChangeOperations.append(ContentChangeOperation(type: type, indexSet: IndexSet(integer: sectionIndex)))
+    }
+
+    func controller(_ controller: NSFetchedResultsController<NSFetchRequestResult>, didChange anObject: Any, at indexPath: IndexPath?, for type: NSFetchedResultsChangeType, newIndexPath: IndexPath?) {
+        contentChangeOperations.append(ContentChangeOperation(type: type, indexPath: indexPath, newIndexPath: newIndexPath))
+    }
+
+    // TODO: Update to new
+    func controllerDidChangeContent(_ controller: NSFetchedResultsController<NSFetchRequestResult>) {
+        collectionView!.performBatchUpdates({
+            let collectionView = self.collectionView!
+            for change in self.contentChangeOperations {
+                switch change.context {
+                    case .section:
+                        switch change.type {
+                            case .insert:
+                                collectionView.insertSections(change.indexSet!) // Shit, I don't know what to do
+                            case .delete:
+                                collectionView.deleteSections(change.indexSet!)
+                            case .move:
+                                break
+                            case .update:
+                                break
+                            }
+                    case .object:
+                        switch change.type {
+                            case .insert:
+                                collectionView.insertItems(at: [change.newIndexPath!])
+                            case .delete:
+                                collectionView.deleteItems(at: [change.indexPath!])
+                            case .update:
+                                // No need to update a cell that has not been loaded.
+                                if let cell = collectionView.cellForItem(at: change.indexPath!) {
+                                    self.delegate?.configureCollectionCell(cell, indexPath: change.indexPath!)
+                                }
+                            case .move:
+                                collectionView.deleteItems(at: [change.indexPath!])
+                                collectionView.insertItems(at: [change.newIndexPath!])
+                        }
+                }
+            }
+        }, completion: nil)
+    }
+
+}
+
+extension CollectionViewMultipleResultsControllerDelegateImplementation : UICollectionViewDataSource {
+
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return (resultsController.count == 1) ? 0 : resultsController.count
```

Comment:
So we will not support multiple sections resultControllers?
If yes, it has to be documented.